Gemfile.lock

```diff
@@ -216,7 +216,7 @@ GEM
       thor (~> 0.14)
     formatador (0.2.5)
     foundation_emails (2.2.0.0)
-    geocoder (1.3.7)
+    geocoder (1.4.0)
```

Comment:
is it possible to update ahoy without updating this. If so, I think it would be better to do it separately to ensure we have a good git history for if we need to do a git bisect or something. This is a handy tool for updating gems in separate commits automatically: https://github.com/lpender/bummr